recipe/meta.yaml

```
@@ -39,7 +37,7 @@ about:
   summary: WebSocket client for Python with low level API options
   dev_url: https://github.com/websocket-client/websocket-client/
   doc_url: https://websocket-client.readthedocs.io/en/latest/
-  license: Apache-2.0
+  license: LGPL-2.1-or-later
   license_file: LICENSE
```

```suggestion
  license_file: 
    - LICENSE
    - COPYING.LESSER
```

Not sure if this being distributed but would need to be included.

Ref: https://github.com/websocket-client/websocket-client/blob/v1.0.0/COPYING.LESSER